setup.py

```diff
@@ -10,18 +10,19 @@
 
 
 requirements = [
-    # TODO(sjperkins)
-    # Depend on actual releases once this branch is merged
     'codex-africanus[dask] >= 0.3.2',
-
-    # TODO(sjperkins)
-    # Depend on actual releases
-    'dask-ms >= 0.2.6',
-
+    # breakage dask-ms 0.2.12
+    # issue 254
```

Comment:
can possibly just change to != if we can find a fix

Reply:
https://github.com/ratt-ru/dask-ms/pull/255 should fix this.

Would you like a 0.2.13 release?

Reply:
Released 0.2.14